configs/emulationstation/es_settings.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <string name="ROMDirectory" value="/run/media/mmcblk0p1/Emulation/roms/" />
-<string name="MediaDirectory" value="/run/media/mmcblk0p1/Emulation/storage/download_media" />
```

Comment:
Independent of this PR being accepted, I believe there is a typo here of `downloaded_media`